### DISCUSSION TITLE
Can I access command line `string[] args` array on Windows in MAUI?

### DISCUSSION THREAD

**Discussion by MartyIX** [dotnet/maui#5413] [Q&A] [answered]
Hi

is it possible to get `string[] args` array containing command-line arguments used to run my MAUI application as one can get in an ordinary C# console application?

Thanks!

**Comment by nogginbox** [comment]
I've not tried this, but it looks like you can in the `OnLaunched` method which is passed `LaunchActivatedEventArgs`.

Here are the [docs for the OnLaunched method](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.application.onlaunched?view=winui-3.0).

**Comment by clamchoda** [comment, marked as answer]
You can also call  `Environment.GetCommandLineArgs();`

**Reply by MartyIX** [reply]
Thank you!
